docs/reference/tctl.md

```diff
@@ -7,42 +7,31 @@ tags:
   - tctl
 ---
 
-import RelatedReadList from '../components/RelatedReadList.js'
-
-<!-- prettier-ignore -->
-import * as TctlWorkflow from './tctl/workflow.md'
-import * as TctlActivity from './tctl/activity.md'
-import * as TctlTaskqueue from './tctl/taskqueue.md'
-import * as TctlBatch from './tctl/batch.md'
-
-import WhatIsTctl from '/docs/content/what-is-tctl.md'
-
-<WhatIsTctl/>
+[What is tctl?](/docs/content/what-is-tctl)
 
 ## tctl commands
 
-- tctl namespace
-- <preview page={TctlWorkflow}>tctl workflow</preview>
-- <preview page={TctlActivity}>tctl activity</preview>
-- <preview page={TctlTaskqueue}>tctl taskqueue</preview>
-- <preview page={TctlBatch}>tctl batch</preview>
-- tctl cluster
-- tctl dataconverter
-- tctl help
-- tctl admin cluster
-- tctl admin db
-- tctl admin decode
-- tctl admin dlq
-- tctl admin elasticsearch
-- tctl admin history_host
-- tctl admin membership
-- tctl admin namespace
-- tctl admin shard
-- tctl admin taskqueue
-- tctl admin workflow
-
-## tctl environment variables
-
-import EnvironmentVariables from './tctl/environment-variables.md'
-
-<EnvironmentVariables/>
+- `tctl namespace`
+- [`tctl workflow`](/docs/reference/tctl/workflow)
+- [`tctl activity`](/docs/reference/tctl/activity)
+- [`tctl taskqueue`](/docs/reference/tctl/taskqueue)
+- [`tctl batch`](/docs/reference/tctl/batch)
+- `tctl cluster`
+- `tctl dataconverter`
+- `tctl help`
+- `tctl admin cluster`
+- `tctl admin db`
+- `tctl admin decode`
+- `tctl admin dlq`
+- `tctl admin elasticsearch`
+- `tctl admin history_host`
+- `tctl admin membership`
+- `tctl admin namespace`
+- `tctl admin shard`
+- `tctl admin taskqueue`
+- `tctl admin workflow`
+
+## Related
+
+[How to install tctl](/docs/content/how-to-install-tctl)  
+[Environment variables for tctl](./tctl/environment-variables.md)
```

Comment:
Use a `/docs` url here for consistency (mostly for search and replace later on ;) )